base/gmp.jl

```diff
@@ -84,8 +84,10 @@ function __init__()
     try
         if gmp_version().major != GMP_VERSION.major || gmp_bits_per_limb() != GMP_BITS_PER_LIMB
             msg = gmp_bits_per_limb() != GMP_BITS_PER_LIMB ? error : warn
-            msg(string("The dynamically loaded GMP library (version $(gmp_version()) with __gmp_bits_per_limb == $(gmp_bits_per_limb()))\n",
-                       "does not correspond to the compile time version (version $GMP_VERSION with __gmp_bits_per_limb == $GMP_BITS_PER_LIMB).\n",
+            msg(string("The dynamically loaded GMP library (version $(gmp_version()),
+                       with __gmp_bits_per_limb == $(gmp_bits_per_limb()))\n",
+                       "does not correspond to the compile time version",
```

Comment:
doesn't look like it was? either this line should have a space before the close quote or the next line should have a space before the open quote